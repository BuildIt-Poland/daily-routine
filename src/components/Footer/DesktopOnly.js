import styled from 'styled-components';

import media from '../../styles/media';

const DesktopOnly = styled.span`
  ${media.phone`
    display: none;
  `}
`;

export default DesktopOnly;
