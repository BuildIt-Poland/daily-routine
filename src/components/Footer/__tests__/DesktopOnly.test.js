import React from 'react';
import { create } from 'react-test-renderer';

import DesktopOnly from '../DesktopOnly';

describe('COMPONENT - Footer DesktopOnly', () => {
  it('renders correctly', () => {
    const component = create(<DesktopOnly />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
