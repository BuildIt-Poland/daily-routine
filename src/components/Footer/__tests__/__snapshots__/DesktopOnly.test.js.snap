// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Footer DesktopOnly renders correctly 1`] = `
@media (max-width:26.25em) {
  .c0 {
    display: none;
  }
}

<span
  className="c0"
/>
`;
